### PR TITLE
BZ1951185 KVM on IBM Z Install with static ip addresses

### PR DESCRIPTION
**Pull request by SNiemann15**
- BZ https://bugzilla.redhat.com/show_bug.cgi?id=1951185
- JIRA issue: https://issues.redhat.com/browse/MULTIARCH-973
- OCP version for cherry-picking: 4.7, 4.8
- Reviewer: Wolfgang Voesch, Holger Wolf
- G-doc draft: https://docs.google.com/document/d/1Gz4gYWwyGqxRj9mVVLeL-nSHW71cdBk_ajCqQBz9eGo/edit#heading=h.8rhprnlv5zrr
- [OpenShift New Feature Enablement charts for P/Z](https://docs.google.com/presentation/d/1TJMz2AFqkWUB7BTGuArkWpeomrjT2aSBdTVnfwQnfEU/edit#slide=id.g5a662668d9_0_5)
- Preview
   -  [Creating the user provisioned infrastructure](https://deploy-preview-31704--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-ibm-z-kvm.html#installation-infrastructure-user-infra_installing-ibm-z-kvm)
   - [Creating Red Hat Enterprise Linux CoreOS (RHCOS) machines](https://deploy-preview-31704--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-ibm-z-kvm.html#installation-user-infra-machines-iso-ibm-z_kvm_installing-ibm-z-kvm)

### CODE REVIEW

**Comment by alkleinrh** [comment]
lgtm

**Comment by wvoesch** [comment]
lgtm

**Comment by jboxman** [comment]
@SNiemann15, so can the two installation paths on this be completely separate modules and included in the assembly? _Choose the installation method you want to follow to deploy Red Hat Enterprise Linux CoreOS (RHCOS):_ is not so much of a prerequisite as a decision to make about which procedure to follow.

I realize this is a moderate amount of rework and probably not what you want to hear, but it would be great if you'd consider separating these into distinct procedure modules with an indication that a user needs to only complete one or the other.

Thanks!

File: modules/installation-infrastructure-user-infra.adoc [hunk 1]
```diff
@@ -35,6 +42,9 @@ endif::ibm-z[]
 ifndef::ibm-z[]
 . Configure DHCP or set static IP addresses on each node.
 endif::ibm-z[]
+ifdef::ibm-z-kvm[]
```
Comment (SNiemann15) [inline modules/installation-infrastructure-user-infra.adoc:45]: @jboxman 
Trying to figure out how to best approach your comment how about changing this too: 
`. Choose to perform a either a fast track installation of {op-system-first}  or a full installation {op-system-first} For the full installation you must set up an HTTP or HTTPS server to provide Ignition files to the cluster nodes. This is not required for a fast track installation. See sections “Creating {op-system-first} machines - fast track installation" and “Creating {op-system-first} machines - full installation".`

And then add a two separate  modules to the KVM assembly.

Comment (jboxman) [reply modules/installation-infrastructure-user-infra.adoc:45]: @SNiemann15, sure that seems reasonable, thanks!

**Comment by SNiemann15** [comment]
@jboxman I've restructured can you PTAL? If you say its ok now I'll ask my peers to QE review again.

**Comment by holgwolf** [comment]
LGTM

**Comment by jboxman** [comment]
> @jboxman I've restructured can you PTAL? If you say its ok now I'll ask my peers to QE review again.

@SNiemann15, for the module titles, is something like the following possible:

> Fast-track installation: Creating…
> Full installation: Creating…

**Comment by SNiemann15** [comment]
> > @jboxman I've restructured can you PTAL? If you say its ok now I'll ask my peers to QE review again.
> 
> @SNiemann15, for the module titles, is something like the following possible:
> 
> > Fast-track installation: Creating…
> > Full installation: Creating…

@jboxman: Done. 
Actually I had it like that in my first local draft then decided to do it the other way round :-)

**Pull request merged**